osu.Server.Queues.ScoreStatisticsProcessor/Commands/Maintenance/VerifyImportedScoresCommand.cs

```diff
@@ -81,7 +81,12 @@ public async Task<int> OnExecuteAsync(CancellationToken cancellationToken)
 
                     if (importedScore.HighScore == null)
                     {
-                        // TODO: delete imported score
+                        await conn.ExecuteAsync("DELETE FROM scores WHERE id = @newId; DELETE FROM score_performance WHERE score_id = @newId; DELETE FROM solo_scores_legacy_id_map WHERE old_score_id = @oldId", new
```

Comment:
ummmmmm wasn't `solo_scores_legacy_id_map` the wrong one and now it's `scores_legacy_id_map`...?

Reply:
yep 😵‍💫

Reply:
was *proooobably* lifted from

https://github.com/ppy/osu-queue-score-statistics/blob/e4e243313f8d7bf09e31453a6952aabc0f9f241d/osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/BatchInserter.cs#L108

and we probably wanna fix that one too x_x

Reply:
Yeah well fuck

Reply:
Also it's missing a mode specification. Which will completely obliterate wrong scores in the worst case scenario.

Reply:
I'm not seeing mode specs in d9cf7611f2f15639a657345ae9cbf82421cc7105 or b6ac0c900e49a7248323167b3e44e6b1c58786e6, is that something yet to be added still?

Reply:
I'll push that now. Was testing on another branch.

Reply:
I had this marked for cleanup requirements, but the table doesn't exist anymore so that is no longer required.